classes/Context.php

```diff
@@ -391,6 +392,16 @@ public function getTranslator($isInstaller = false)
         return $this->translator;
     }
 
+    /**
+     * @param string $locale
+     * @param string $notName
+     */
+    public function addLanguageToTranslator($locale, $notName = null)
```

Comment:
It seems `$notName` is not really useful here (you don't use it in the `Language` class)